source/valohai-yaml/pipeline/index.rst

```diff
@@ -39,31 +39,47 @@ A simple pipeline could look something like this:
     - pipeline:
         name: simple-pipeline
         nodes:
-          - name: generate
+          - name: generate-node
             type: execution
             step: generate-dataset
-          - name: train
+          - name: train-node
             type: execution
             step: train-model
+          - name: deploy-node
+            type: deployment
+            deployment: predict-digit
```

Comment:
Should we name this something else to make it clear that it is the name of the deployment, not the same as the endpoint? To prevent confusion.

Reply:
Yep - that's a mistake. Changed to `mydeployment`